Extension/browser/firefox/lib/elemHide.js

```diff
@@ -14,30 +14,28 @@
  * You should have received a copy of the GNU Lesser General Public License
  * along with Adguard Browser Extension.  If not, see <http://www.gnu.org/licenses/>.
  */
-var {Cc, Ci, Cu} = require('chrome');
+
+/* global Services, require */
+
+var {Ci, Cu} = require('chrome');
 
 Cu.import("resource://gre/modules/Services.jsm");
 Cu.import("resource://gre/modules/FileUtils.jsm");
 
 var self = require('sdk/self');
 
-var {EventNotifier} = require('./utils/notifier');
-let {FS} = require('./utils/file-storage');
-var {EventNotifierTypes} = require('./utils/common');
-var {ConcurrentUtils} = require('./utils/browser-utils');
-var {Log} = require('./utils/log');
-var {userSettings} = require('./utils/user-settings');
+var ConcurrentUtils = require('./utils/browser-utils').ConcurrentUtils;
+var Log = require('./utils/log').Log;
 var Prefs = require('./prefs').Prefs;
 var styleService = require('./styleSheetService');
 
 /**
- * This object manages CSS and JS rules.
+ * This object manages CSS styles
  *
- * Depending on the user settings we can use one of the following ways:
- * 1. Registering browser-wide stylesheet
- * 2. Injecting CSS/JS with content-script/preload.js script
+ * 1. Registering assistant style
+ * 2. Registering collapse elements style

```

Comment:
I suppose we can get rid of point 2 as well.
`collapse_by_content_script` is true by default anyway.